package.json

```diff
@@ -68,56 +68,56 @@
 		}
 	},
 	"dependencies": {
-		"@babel/core": "^7.12.10",
-		"@babel/plugin-proposal-class-properties": "7.12.1",
+		"@babel/core": "^7.24.6",
 		"@babel/plugin-syntax-import-meta": "^7.10.4",
-		"@babel/plugin-syntax-jsx": "^7.12.1",
-		"@babel/plugin-transform-flow-strip-types": "^7.12.10",
-		"@babel/plugin-transform-react-jsx": "^7.12.11",
-		"@babel/plugin-transform-regenerator": "^7.12.1",
-		"@babel/preset-env": "^7.12.11",
-		"@babel/preset-flow": "^7.12.1",
-		"@babel/preset-react": "^7.12.10",
-		"@rollup/plugin-alias": "^3.1.1",
-		"@rollup/plugin-babel": "^5.2.2",
-		"@rollup/plugin-commonjs": "^17.0.0",
+		"@babel/plugin-syntax-jsx": "^7.24.6",
+		"@babel/plugin-transform-class-properties": "7.24.6",
+		"@babel/plugin-transform-flow-strip-types": "^7.24.6",
+		"@babel/plugin-transform-react-jsx": "^7.24.6",
+		"@babel/plugin-transform-regenerator": "7.24.6",
+		"@babel/preset-env": "^7.24.6",
+		"@babel/preset-flow": "^7.24.6",
+		"@babel/preset-react": "^7.24.6",
+		"@rollup/plugin-alias": "^5.1.0",
+		"@rollup/plugin-babel": "^6.0.4",
+		"@rollup/plugin-commonjs": "^25.0.8",
 		"@rollup/plugin-json": "^4.1.0",
-		"@rollup/plugin-node-resolve": "^11.0.1",
-		"@surma/rollup-plugin-off-main-thread": "^2.2.2",
+		"@rollup/plugin-node-resolve": "^15.2.3",
+		"@surma/rollup-plugin-off-main-thread": "^2.2.3",
 		"asyncro": "^3.0.0",
-		"autoprefixer": "^10.1.0",
-		"babel-plugin-macros": "^3.0.1",
+		"autoprefixer": "^10.4.19",
+		"babel-plugin-macros": "^3.1.0",
 		"babel-plugin-transform-async-to-promises": "^0.8.18",
 		"babel-plugin-transform-replace-expressions": "^0.2.0",
 		"brotli-size": "^4.0.0",
-		"builtin-modules": "^3.1.0",
-		"camelcase": "^6.2.0",
+		"builtin-modules": "^3.2.0",
+		"camelcase": "^6.3.0",
 		"escape-string-regexp": "^4.0.0",
-		"filesize": "^6.1.0",
+		"filesize": "^8.0.7",
 		"gzip-size": "^6.0.0",
-		"kleur": "^4.1.3",
+		"kleur": "^4.1.5",
 		"lodash.merge": "^4.6.2",
 		"postcss": "^8.2.1",
-		"pretty-bytes": "^5.4.1",
-		"rollup": "^2.35.1",
+		"pretty-bytes": "^5.6.0",
+		"rollup": "^4.18.0",
```

Comment:
Looks like `rollup` is up to `4.29.1` now:

- https://github.com/rollup/rollup/blob/master/CHANGELOG.md#4180